Tests/registrationTests.js

```diff
@@ -0,0 +1,61 @@
+var homePage = require('../Pages/homePage.js'),
+    itemPage = require ("../Pages/itemPage"),
+    registerPage = require ("../Pages/registerPage"),
+    data = require("../Data/data.js");
+    
+browser.waitForAngularEnabled(false);
+browser.ignoreSynchronization = true;
+browser.manage().window().maximize();
+
+describe("002: Registration", function(){
+    beforeEach(() => {
+        homePage.openPageURL(data.homepageLink)
+        .then(() => homePage.clickOnLinks(data.createAnAccountTitle))
+    })
+
+    it("001: User is able to create account with valid credentials", function(){
+        registerPage.registrateAnAccount(data.firstNameTester,data.lastNameTester,data.emailTitle,data.passwordTester,data.passwordTester)
+            .then(() => itemPage.waitForElement(data.successMessage))
+            .then(() => registerPage.validateInputMessages())    
+    }) 
+    it("002: User isn't able to create account with invalid credentials", function(){
+        registerPage.registrateAnAccount(data.fiveSpaces,data.fiveSpaces,data.emailTitle,data.fiveSpaces,data.fiveSpaces)
+            .then(() => registerPage.validateInputMessages())
```

Comment:
Since this part "  registerPage.registrateAnAccount(data.fiveSpaces,data.fiveSpaces,data.emailTitle,data.fiveSpaces,data.fiveSpaces)
            .then(() => registerPage.validateInputMessages())" is repeating 9 times can you make one uiTask from it ?

Reply:
I added .then(() => registerPage.validateInputMessages())" as a part of the registrateAnAccount() method.